db/migration/202009171251-init-database.go

```diff
@@ -0,0 +1,89 @@
+package migration
+
+import migrate "github.com/rubenv/sql-migrate"
+
+func Get202009171251() *migrate.Migration {
+	return &migrate.Migration{
+		Id: "123",
```

Comment:
I think for minimizing context it would be best to have the Id match the name?

Reply:
yep

Reply:
It would be great to have a sed or similar check that Id matches `Get.*`.